optimism.tokenlist.json

```diff
@@ -1569,6 +1569,28 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x07FA101efde726E0956Edd2c4D5C8d3d1A5E9c53",
+      "name": "Respawn Finance Wrapped Staked Ethereum",
+      "symbol": "RFWSTETH",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/RFWSTETH.png",
+      "extensions": {
+        "optimismBridgeAddress": "0xeAeAdAC73baaF4cB8B024dE9D65B2eeFa722856C"
```

Comment:
This is not the bridge address. The bridge address from L1 is [0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1](https://etherscan.io/address/0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1#code0. This is why CI is failing.

Reply:
Additionally, you are not providing an Optimism token address (chainId 10) but you are specifying a bridge address. Thus CI is failing because it is trying to match your L1 contract with your unspecified L2 contract.